engine-precompiles/src/native.rs

```diff
@@ -216,6 +216,13 @@ impl<I> ExitToNear<I> {
     }
 }
 
+fn is_valid_input_size(input: &[u8], min: usize, max: Option<usize>) -> Result<(), ExitError> {
+    if max.is_some() && (input.len() < min || input.len() > max.unwrap()) {
```

Comment:
Probably more rusty: 
```suggestion
    if input.len() < min || map.map_or(false, |max| input.len() > max)) {
```

Reply:
And in your version you don't check `min` lenght if `max` is None. Add test case: 
```rust
#[test]
     fn test_check_valid_input() {
         let input = [1u8; 1];
         is_valid_input_size(&input, 2, None).unwrap();
     }
```
This test would pass.

Reply:
it is failing here https://github.com/aurora-is-near/aurora-engine/actions/runs/4145399283/jobs/7169697666#step:9:375